lang/strings_english.txt

```diff
@@ -1192,7 +1192,7 @@ $s_average_time = 'Average time';
 $s_total_time = 'Total time';
 $s_developer_stats = 'Developer Stats';
 $s_reporter_stats = 'Reporter Stats';
-$s_orct = '(open/resolved/closed/total)';
+$s_orct = '(open/resolved/closed/total/resolved ratio/ratio)';
```

Comment:
Will this break for non-translated languages?  Should we create a new string and delete this one?

Reply:
Indentation fixed.  I am not sure about the translation process. While using the old string name, in the non-translated languages the last two new headers in summary tables will not be shown , but the two new columns values will be shown.  If  we create a new string, no headers will be shown in non-translated languages at all.  Please make a decision, after than I will adjust the string file and codes.

Reply:
Correct: If we create a new string, only English strings headers will be shown in non-translated languages.

Reply:
Rename the string variable and keep the old one maybe is the better way. I searched the code tree and found   MantisGraph referenced $s_orct  string at line 130, but not used that actually in other codes.
./plugins/MantisGraph/pages/issues_trend_bystatus_table.php:$t_label_string = lang_get( 'orct' ); # use the (open/resolved/closed/total) label